spec/spec.md

```diff
@@ -0,0 +1,174 @@
+# Verona Language Spec
+
+## Lexical Elements
+
+### Comments
+
+### Identifiers and Symbols
+
+- `_` as "don't care".
+
+### Strings
+
+### Constants
+
+- Boolean
+- Various integers and floats
+
```

Comment:
are strings not constant?

Reply:
Perhaps call the section "String literals" and move after "object literals"

Reply:
Do we have "array literals"?